.github/workflows/ci.yaml

```diff
@@ -619,10 +620,13 @@ jobs:
           java-version: '8'
           distribution: 'temurin'
 
+      - name: Create no superuser
+        run: ./ci/no-superuser/create-no-superuser-sqlserver.sh sqlserver17 SqlServer17
```

Comment:
Use a new script to create `no superuser`.

Reply:
Sorry, it's a super minor thing. In this case, is a "non-superuser" correct grammatically? Although PostgreSQL has an option "--no-superuser" since it's an option name, I think we say basically "non-root" user or something like that. What do you think?

Reply:
Ah, yes. You're right. Actually, I was struggling with that too...

In our (ScalarDB SQL's) SQL Grammar syntax, we also use the term `NO_SUPERUSER` same as the PostgreSQL option name to indicate the `non-root` user.
https://scalardb.scalar-labs.com/docs/latest/scalardb-sql/grammar/#create-user

So, to make documents and codes consistent, I chose `no superuser` in this update. However, I agree with you. Usually, I think `non-superuser` is correct from the perspective of English grammar.

So, I think we have to decide which is important. `Consisntecy in documents and codes` or `English grammar correctness`.
(As I mentioned, I chose the `Consisntecy in documents and codes`. This is because we support some kinds of databases. We cannot follow the terms of one specific database. So, I decided to follow the term of ScalarDB SQL document.)

What do you think?

Reply:
Oh, I didn't know we had already used the term "no_superuser"... In that case, it's one of the reasonable choices to make them consistent. Thank you!

Reply:
I also think `no_superuser` is pretty weird as an English, but we decided to follow PostgreSQL.